web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,170 @@
+                        <span class="pf-c-switch__toggle-icon">
+                            <i class="fas fa-check" aria-hidden="true"></i>
+                        </span>
+                    </span>
+                    <span class="pf-c-switch__label">${msg("Execution logging")}</span>
+                </label>
+                <p class="pf-c-form__helper-text">
+                    ${msg(
+                        "When this option is enabled, all executions of this policy will be logged. By default, only execution errors are logged.",
+                    )}
+                </p>
+            </ak-form-element-horizontal>
+            <ak-form-group .expanded=${true}>
+                <span slot="header"> ${msg("Policy-specific settings")} </span>
+                <div slot="body" class="pf-c-form">
+                    <ak-form-element-horizontal label=${msg("ASN Mode")} name="asnMode">
+                        <ak-search-select
+                            .fetchObjects=${async () => {
+                                return [
+                                    { name: "allow", label: "Allow" },
+                                    { name: "block", label: "Block" },
+                                ];
+                            }}
+                            .renderElement=${(item: TypeCreate): string => {
+                                return item.label;
+                            }}
+                            .value=${(item: TypeCreate | undefined): string | undefined => {
+                                return item.name;
+                            }}
+                            .selected=${(item: TypeCreate): boolean => {
+                                return this.instance?.asnMode === item.name;
+                            }}
+                        >
+                        </ak-search-select>
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "Block or only allow connections from IPs from the provided autonomous systems.",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("ASN List")} name="asnList">
+                        <input
+                            type="text"
+                            value="${ifDefined(this.instance?.asnList || "")}"
+                            class="pf-c-form-control"
+                        />
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "List of autonomous system numbers. Comma separated. E.g. 13335,15169,20940",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("Country Mode")} name="countryMode">
+                        <ak-search-select
```

Comment:
... and given that you've done it twice, break it out into a variable or something so you'll only have to change it once if the rules change.  :-)